translations/en_GB/atrium-translations-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionPathAssertion.kt

```diff
@@ -40,4 +41,5 @@ enum class DescriptionPathAssertion(override val value: String) : StringBasedTra
     HINT_FOLLOWED_SYMBOLIC_LINK("followed the symbolic link %s to %s"),
     HAS_SAME_TEXTUAL_CONTENT("has same textual content with encoding %s, %s"),
     HAS_SAME_BINARY_CONTENT("has same binary content"),
+    RELATIVE_PATH("relative path"),
```

Comment:
```suggestion
    RELATIVE_PATH("a relative path"),
```

If I read the code correct, the failure message will currently read “is relative path”, which is missing an article.